product_docs/docs/postgres_distributed_for_kubernetes/1/architecture.mdx

```diff
@@ -192,3 +192,8 @@ reliably communicate with each other.
 can be realized on multiple Kubernetes clusters that meet the connectivity
 requirements.
 More information can be found in the ["Connectivity"](connectivity.md) section.
+
+!!! Note Availability zones 
```

Comment:
Should this not be regions and availability zones? (Given that one region can have multiple availability zones but the liklihood is that cross-region deployments are whats on peoples minds.)